src/py2spack/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""py2spack package.
+
+This package allows converting standard python packages to Spack packages.
+"""
```

Comment:
The first line should be the summary, not  the name of the package (which is already encoded in the folder or file name). 
```suggestion
"""A package to convert Python distribution packages to Spack packages."""
```

All comments like this one about general conventions or style rules apply to all files in the repo, even if I will only the mention it the first time they appear.